mdsuite/calculators/radial_distribution_function.py

```diff
@@ -322,12 +352,12 @@ def _load_positions(self, indices: list) -> tf.Tensor:
         loaded_data : tf.Tensor
                 tf.Tensor of tensor_values loaded from the hdf5 database_path
         """
-
-        if self.molecules:
-            path_list = [join_path(species, "Positions") for species in self.experiment.molecules]
-            return self.experiment.load_matrix("Positions", path=path_list, select_slice=np.s_[:, indices])
+        path_list = [join_path(species, "Positions") for species in self.species]
+        data = self.experiment.load_matrix("Positions", path=path_list, select_slice=np.s_[:, indices])
+        if len(self.species) == 1:
+            return tf.concat([data], axis=0)
```

Comment:
`tf.concat([data], axis=0)` == `data`
you should be able to just return data

Reply:
Yes! I have changed it.